Add details of NIC requirements (#543)
harvester/docs: docs/install/requirements.md

| Memory           | 32 GB minimum for testing; 64 GB or above required for production                                                                                                                                                               |
| Disk Capacity    | 250 GB minimum for testing (180 GB minimum when using multiple disks); 500 GB or above required for production                                                                                                                                  |
| Disk Performance | 5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for etcd](https://www.suse.com/support/kb/doc/?id=000020100) |

> | Network Card     | 1 Gbps Ethernet minimum for testing; 10Gbps Ethernet required for production; suggest to use 2 NICs for the management cluster network, 2 additional NICs for the dedicated VM workload network       |

Reviewer: ```suggestion
| Network Card     | Minimum Ethernet speed: 1 Gbps for testing, 10 Gbps for production; Recommended network interfaces: 2 NICs for the management cluster network, 2 additional NICs for the dedicated VM workload network |
```

> | Network Card     | Minimum Ethernet speed: 1 Gbps for testing, 10 Gbps for production; Recommended network interfaces: 2 NICs for the management cluster network, 2 additional NICs for the dedicated VM workload network (except the [Witness Node](../advanced/witness.md))       |

Reviewer: ```suggestion
| Network Card     | Minimum Ethernet speed: 1 Gbps for testing, 10 Gbps for production; Recommended network interfaces: 2 NICs for the management cluster network, 2 additional NICs for the dedicated VM workload network (does not apply to the [witness node](../advanced/witness.md))       |
```

Reviewer: The requested change was not applied.

| Network Switch   | Trunking of ports required for VLAN support                                                                                                                                                           |

:::info important
- Use server-class hardware to achieve the best results. Laptops and nested virtualization are not supported.
- Each node must have a unique `product_uuid` (fetched from `/sys/class/dmi/id/product_uuid`) to prevent errors from occurring during VM live migration and other operations. For more information, see [Issue #4025](https://github.com/harvester/harvester/issues/4025).

> - Harvester has a [built-in management cluster network](../networking/clusternetwork.md#built-in-cluster-network). To achieve the best performance and high-availability. The general practice in production is to use at least 2 NICs in each node to setup a bonded NIC for the management network. Refer the step 6 of the [ISO Installation](../install/iso-install.md#installation-steps). You may also create additional [customized cluster network](../networking/clusternetwork.md#custom-cluster-network) to carry the VM workload network separately. Each of them requires at least 2 additional NICs to setup a bonded NIC in every involved node of the Harvester cluster. You may refer [Cluster Network](../networking/clusternetwork.md#concepts) for more details.

Reviewer 2: > To achieve the best performance and high-availability.

This sentence seems to be incomplete.

> - Harvester has a [built-in management cluster network](../networking/clusternetwork.md#built-in-cluster-network). To achieve the best performance and high-availability. The general practice in production is to use at least 2 NICs in each node to setup a bonded NIC for the management network. Refer the step 6 of the [ISO Installation](../install/iso-install.md#installation-steps). You may also create additional [customized cluster network](../networking/clusternetwork.md#custom-cluster-network) to carry the VM workload network separately. Each of them requires at least 2 additional NICs to setup a bonded NIC in every involved node of the Harvester cluster. You may refer [Cluster Network](../networking/clusternetwork.md#concepts) for more details.

Reviewer: ```suggestion
- Harvester has a [built-in management cluster network](../networking/clusternetwork.md#built-in-cluster-network) (`mgmt`). To achieve high availability and the best performance in production environments, use at least two NICs in each node to set up a bonded NIC for the management network (see step 6 in [ISO Installation](../install/iso-install.md#installation-steps)). You can also create [custom cluster networks](../networking/clusternetwork.md#custom-cluster-network) for VM workloads. Each custom cluster network requires at least two additional NICs to set up a bonded NIC in every involved node of the Harvester cluster. For more information, see [Cluster Network](../networking/clusternetwork.md#concepts).
```

> - For testing purpose, you may use only one NIC for the [built-in management cluster network](../networking/clusternetwork.md#built-in-cluster-network), and even to test the [VM network](../networking/harvester-network.md#create-a-vm-network) which is also carried by this [built-in management cluster network](../networking/clusternetwork.md#built-in-cluster-network). There is no assurance of performance or high-availability.

Reviewer: ```suggestion
- During testing, you can use only one NIC for the [built-in management cluster network](../networking/clusternetwork.md#built-in-cluster-network) (`mgmt`), and for testing the [VM network](../networking/harvester-network.md#create-a-vm-network) that is also carried by `mgmt`. High availability and optimal performance are not guaranteed.
```
